postreise/plot/tests/test_plot_pie_generation_vs_capacity.py

```diff
@@ -50,7 +50,7 @@ def test_handle_plot_inputs_throws_typeError_for_resource_labels_not_in_a_dictio
         )
 
 
-def test_handle_plot_inputs_throws_typeError_for_resource_colors_not_in_a_dictionary_format():
+def test_plot_pie_generation_vs_capacity_throws_typeError_for_resource_colors_not_in_a_dictionary_format():
     with pytest.raises(TypeError):
         plot_pie_generation_vs_capacity(
             areas=["all", "Western", "Texas", "Eastern"],
```

Comment:
same comment regarding the the naming of the function with `typeError` and `valueError`

Reply:
Done.